charts/oidc-webhook-authenticator/charts/runtime/templates/deployment.yaml

```
@@ -11,6 +11,11 @@ metadata:
     app.kubernetes.io/name: {{ include "oidc-webhook-authenticator.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- if .Values.additionalLabels.deployment }}
+{{- range .Values.additionalLabels.deployment }}
+{{ toYaml . | trim | indent 4 }}
+{{- end }}
+{{- end }}
```

If you accept to change the labels from list to map, this can be simplified
```suggestion
{{- if .Values.additionalLabels.deployment }}
{{ toYaml .Values.additionalLabels.deployment | trim | indent 4 }}
{{- end }}
```

Also, can you add these labels to the pod template? There might be services, network policies, etc. that can try to use these additional labels via pod selectors, but it will not work as the pod will not have them.

---

> Also, can you add these labels to the pod template? There might be services, network policies, etc. that can try to use these additional labels via pod selectors, but it will not work as the pod will not have them.

What do you think about this?

---

Yes, this can be a good addition. I added it to my last commit.